input/fsh/valuesets/au-erequesting-coverage-self-pay.fsh

```diff
@@ -0,0 +1,16 @@
+ValueSet: AUeRequestingCoverageSelfPay
```

Comment:
Commenting here to get an inline comment. 
Please add to the Change Log.

Reply:
Updated log to add new section for AU Coverage